### ISSUE TITLE
[Bug]: react-router-dom-v5-compat@6.4.1 peer dependency issue

### ISSUE THREAD

**Issue by antonio7905** [remix-run/react-router#9336]
### What version of React Router are you using?

v6.4.1

### Steps to Reproduce

Install `react-router-dom@5` then install  `react-router-dom-v5-compat`.

```bash
node --version
# v16.17.0

npm --version
# 8.15.0

npm install react-router-dom@5

npm install react-router-dom-v5-compat@6.4.1
```

### Expected Behavior

Set peer dependency to `"react-router-dom": "4 || 5"` instead of `"react-router-dom": "6.4.1"`

### Actual Behavior

Got the following errors while installing `react-router-dom-v5-compat`:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: react-router-playground@1.0.0
npm ERR! Found: react-router-dom@5.3.3
npm ERR! node_modules/react-router-dom
npm ERR!   react-router-dom@"^5.3.3" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react-router-dom@"6.4.1" from react-router-dom-v5-compat@6.4.1
npm ERR! node_modules/react-router-dom-v5-compat
npm ERR!   react-router-dom-v5-compat@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
```

**Comment by timdorr** [comment]
Just fixed this in 174897bdd4d626066d67328773ef9b55610c04f9. This will be in the next release.

**Comment by HerrBertling** [comment]
Hey @timdorr :wave: Do you have an ETA for the release? Or should I `--force` the installation and see if things blow up? 😅  I'm not in a rush, but would be great to know 😊

**Comment by timdorr** [comment]
No idea, but @brophdawg11 looks to be putting together some pre-releases for 6.4.2 soon.

**Comment by brophdawg11** [comment]
Yep I think this should be resolved with 6.4.2 - please let us know if it's not 👍
